app/style/app.css

```diff
@@ -133,6 +134,10 @@ gaia-header #upload-link {
   transform: translateY(0);
 }
 
+#app-details #close-button {
+  left: -10px;
```

Comment:
It's the icons in the header. Amy wants something different from what happens by default with the web components (ie. she wants the edge of the icons to line up with the buttons and icons in the app list below).